firebase-firestore/src/main/java/com/google/firebase/firestore/core/Target.java

```diff
@@ -318,12 +318,14 @@ public String getCanonicalId() {
 
     if (startAt != null) {
       builder.append("|lb:");
-      builder.append(startAt.canonicalString());
+      builder.append(startAt.isInclusive() ? "b:" : "a:");
```

Comment:
optional: move this ternary into a method on `Bound`.

Reply:
The result is flipped depending on whether this is a startAt or endAt cursor, hence I pulled it out.